datasquare_test.go

```diff
@@ -67,6 +67,28 @@ func TestRoots(t *testing.T) {
 	}
 }
 
+func TestLazyRootGeneration(t *testing.T) {
+	square, err := newDataSquare([][]byte{{1}, {2}, {3}, {4}}, NewDefaultTree)
+	if err != nil {
+		panic(err)
+	}
+
+	var rowRoots [][]byte
+	var colRoots [][]byte
+
+	for i := uint(0); i < 2; i++ {
```

Comment:
make `2` a var that has a meaningful name.

Reply:
forgot which PR this snuck into but fixed https://github.com/lazyledger/rsmt2d/blob/06294d583ac8f33a01085da3f578ba40f4151c2a/datasquare_test.go#L79